From 088792390fe4a7948053d5bf249500a4b831b1d7 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Sat, 3 Aug 2019 12:46:48 +0300
Subject: [PATCH 21/21] cl-som-imx8: dts: Fix the csi assigned clock rate.

20MHz does not work.
Last good version with that settings is 4.9.88.

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 arch/arm64/boot/dts/compulab/sb-imx8.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/compulab/sb-imx8.dtsi b/arch/arm64/boot/dts/compulab/sb-imx8.dtsi
index 1c9f946..d78368d 100644
--- a/arch/arm64/boot/dts/compulab/sb-imx8.dtsi
+++ b/arch/arm64/boot/dts/compulab/sb-imx8.dtsi
@@ -131,7 +131,7 @@
 		clock-names = "csi_mclk";
 		assigned-clocks = <&clk IMX8MQ_CLK_CLKO2>;
 		assigned-clock-parents = <&clk IMX8MQ_SYS2_PLL_200M>;
-		assigned-clock-rates = <20000000>;
+		assigned-clock-rates = <25000000>;
 		csi_id = <0>;
 		/*
 		To be looked into & fixed
@@ -139,7 +139,7 @@
 		*/
 		pwn-gpios = <&pca9555 8 GPIO_ACTIVE_LOW>;
 		rst-gpios = <&pca9555 7 GPIO_ACTIVE_HIGH>;
-		mclk = <20000000>;
+		mclk = <25000000>;
 		mclk_source = <0>;
 		port {
 		    ov5640_mipi1_ep: endpoint {
-- 
1.9.1

